src/EnergyPlus/SizingManager.cc

```diff
@@ -770,6 +782,778 @@ namespace SizingManager {
+					}
+				} // end loop over zones on air loop to calculate Evz by zone and find mins
+
+				//calculate Vot for both cooling and heating
+				DataSizing::VotClgBySys( AirLoopNum ) = DataSizing::VouBySys( AirLoopNum ) / DataSizing::EvzMinBySysCool( AirLoopNum );
+				DataSizing::VotHtgBySys( AirLoopNum ) = DataSizing::VouBySys( AirLoopNum ) / DataSizing::EvzMinBySysHeat( AirLoopNum );
+				//the design zone ventilation value is based on the larger of the system-level cooling Vot and/or heating Vot
+				FinalSysSizing( AirLoopNum ).DesOutAirVolFlow = max( VotClgBySys( AirLoopNum ), VotHtgBySys( AirLoopNum ) );
+			} // system OA is autosized and VRP
+			else if ( ( FinalSysSizing( AirLoopNum ).OAAutoSized && SysSizInput( SysSizNum ).SystemOAMethod == SOAM_VRP && DataAirLoop::AirLoopZoneInfo( AirLoopNum ).NumZones == 1 ) ) { // single zone VRP
+				int CtrlZoneNum = DataAirLoop::AirLoopZoneInfo( AirLoopNum ).ActualZoneNumber( 1 );
+				// single zone cooling
+				DataSizing::VotClgBySys( AirLoopNum ) = VbzByZone( CtrlZoneNum ) / FinalZoneSizing( CtrlZoneNum ).ZoneADEffCooling;
+				DataSizing::EvzByZoneCool( CtrlZoneNum ) = FinalZoneSizing( CtrlZoneNum ).ZoneADEffCooling;
+				DataSizing::EvzMinBySysCool( AirLoopNum ) = DataSizing::EvzByZoneCool( CtrlZoneNum );
+				DataSizing::VpsClgBySys( AirLoopNum ) = FinalSysSizing( SysSizNum ).DesCoolVolFlow;
+				DataSizing::VpzClgSumBySys( AirLoopNum ) = DataSizing::VdzClgByZone( CtrlZoneNum );
+				// single zone heating
+				DataSizing::VotHtgBySys( AirLoopNum ) = VbzByZone( CtrlZoneNum ) / FinalZoneSizing( CtrlZoneNum ).ZoneADEffHeating;
+				DataSizing::EvzByZoneHeat( CtrlZoneNum ) = FinalZoneSizing( CtrlZoneNum ).ZoneADEffHeating;
+				DataSizing::EvzMinBySysHeat( AirLoopNum ) = DataSizing::EvzByZoneHeat( CtrlZoneNum );
+				DataSizing::VpsHtgBySys( AirLoopNum ) = DataSizing::VpzMinHtgByZone( CtrlZoneNum );
+				DataSizing::VpzHtgSumBySys( AirLoopNum ) = DataSizing::VpzHtgByZone( CtrlZoneNum );
+
+				//the design zone ventilation value is based on the larger of the system-level cooling Vot and/or heating Vot
+				FinalSysSizing( AirLoopNum ).DesOutAirVolFlow = max( VotClgBySys( AirLoopNum ), VotHtgBySys( AirLoopNum ) );
+				//Fill Xs values for reporting
+				DataSizing::XsBySysCool( AirLoopNum ) = FinalSysSizing( AirLoopNum ).DesOutAirVolFlow / DataSizing::VpsClgBySys( AirLoopNum );
+				DataSizing::XsBySysHeat( AirLoopNum ) = FinalSysSizing( AirLoopNum ).DesOutAirVolFlow / DataSizing::VpsHtgBySys( AirLoopNum );
+
+			} else { // not vrp, zone sum, fill out values that still apply
+				// redo VpzClgSumBySys( AirLoopNum ) with latest values, for reporting
+				DataSizing::VpzClgSumBySys( AirLoopNum ) = 0.0;
+				for ( int zoneNumOnLoop = 1; zoneNumOnLoop <= DataAirLoop::AirLoopZoneInfo( AirLoopNum ).NumZones; ++zoneNumOnLoop  ) {
+					int CtrlZoneNum = DataAirLoop::AirLoopZoneInfo( AirLoopNum ).ActualZoneNumber( zoneNumOnLoop );
```

Comment:
This and next for loop are looping through the same zones. Should have combined calculations into one loop.